webpack.config.viewer.js

```diff
@@ -1,6 +1,14 @@
 const common = require('./webpack.config.common.js');
+const VERSION = require('./package.json').version;
 const createApp = common.createApp;
-module.exports = [
-    createApp('viewer', 'molstar'),
-    createApp('mesoscale-explorer', 'molstar'),
-];
\ No newline at end of file
+module.exports = (env, argv) => {
+    return [
```

Comment:
It's not enough to change this file. It is also required in:
-  `webpack.config.production.js`
- Since you are adding more import file types to `index.ts` for the viewer, can you please also adjust the recently added `build-dev.mjs` to support this?